components/engine_rocks/src/sst.rs

```diff
@@ -118,7 +118,7 @@ impl Iterator for RocksSstIterator {
 }
 
 pub struct RocksSstWriterBuilder {
-    cf: Option<CfName>,
+    cf: Option<String>,
```

Comment:
why change it?

Reply:
Because I want to set it in `delete_all_in_range_cf`